timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResults.vue

```diff
@@ -48,7 +48,7 @@ limitations under the License.
               small
               icon
               @click.stop=""
-              @click="triggeredAnalyzerRuns(activeAnalyzerQueue)"
+              @click="startPolling()"
             >
               <v-icon small>mdi-reload-alert</v-icon>
```

Comment:
I noticed a small bug when the AnalyzerResults on the left page run into the timeout and the `mdi-reload-alert` icon is displayed. Even when I trigger a new analyzer for any timeline, the AnalyzerResults progress does not restart but stays in the timeout.

In this screenshot I started the SSH analyzer after the timeout was triggered:
![image](https://github.com/google/timesketch/assets/99879757/ba53cca3-b1e2-4d2c-9901-2cc54d6c99b1)

Reply:
This should be resolved now, because the loading indicator on the right will disappear after x seconds. I could not artificially reproduce the issue with the timeline loading indicator on the left. Maybe you can check if you can still observe it.